packages/extension-sdk-react/src/components/ExtensionProvider/types.ts

```diff
@@ -24,22 +24,13 @@
 
  */
 
-import type { Looker31SDK, Looker40SDK } from '@looker/sdk'
+import type { Looker40SDK } from '@looker/sdk'
```

Comment:
@bryans99 sure you already have thought of this but it would be great to use ILooker40SDK instead here

Reply:
I can create a follow up PR for this